src/store/metadata/stub/content/editions/additions-to-the-housing-stock_2022-2023.json

```diff
@@ -0,0 +1,163 @@
,"E07000039","E10000018","E07000129","E07000130","E07000131","E07000132","E07000133","E07000134","E07000135","E10000019","E07000136","E07000137","E07000138","E07000139","E07000140","E07000141","E07000142","E10000024","E07000170","E07000171","E07000172","E07000173","E07000174","E07000175","E07000176","E12000005","E06000019","E06000051","E06000021","E06000020","E10000028","E07000192","E07000193","E07000194","E07000195","E07000196","E07000197","E07000198","E07000199","E10000031","E07000218","E07000219","E07000220","E07000221","E07000222","E11000005","E08000025","E08000026","E08000027","E08000028","E08000029","E08000030","E08000031","E10000034","E07000234","E07000235","E07000236","E07000237","E07000238","E07000239","E12000006","E06000055","E06000056","E06000032","E06000031","E06000033","E06000034","E10000003","E07000008","E07000009","E07000010","E07000011","E07000012","E10000012","E07000066","E07000067","E07000068","E07000069","E07000070","E07000071","E07000072","E07000073","E07000074","E07000075","E07000076","E07000077","E10000015","E07000095","E07000096","E07000242","E07000098","E07000099","E07000240","E07000243","E07000102","E07000103","E07000241","E10000020","E07000143","E07000144","E07000145","E07000146","E07000147","E07000148","E07000149","E10000029","E07000200","E07000244","E07000202","E07000203","E07000245","E12000007","E09000007","E09000001","E09000012","E09000013","E09000014","E09000019","E09000020","E09000022","E09000023","E09000025","E09000028","E09000030","E09000032","E09000033","E09000002","E09000003","E09000004","E09000005","E09000006","E09000008","E09000009","E09000010","E09000011","E09000015","E09000016","E09000017","E09000018","E09000021","E09000024","E09000026","E09000027","E09000029","E09000031","E12000008","E06000036","E06000043","E06000060","E06000046","E06000035","E06000042","E06000044","E06000038","E06000039","E06000045","E06000037","E06000040","E06000041","E10000011","E07000061","E07000062","E07000063","E07000064","E07000065","E10000014","E07000084","E07000085","E07000086","E07000087","E07000088","E07000089","E07000090","E07000091","E07000092","E07000093","E07000094","E10000016","E07000105","E07000106","E07000107","E07000108","E07000112","E07000109","E07000110","E07000111","E07000113","E07000114","E07000115","E07000116","E10000025","E07000177","E07000178","E07000179","E07000180","E07000181","E10000030","E07000207","E07000208","E07000209","E07000210","E07000211","E07000212","E07000213","E07000214","E07000215","E07000216","E07000217","E10000032","E07000223","E07000224","E07000225","E07000226","E07000227","E07000228","E07000229","E12000009","E06000022","E06000058","E06000023","E06000052","E06000059","E06000053","E06000024","E06000026","E06000025","E06000030","E06000027","E06000054","E10000008","E07000040","E07000041","E07000042","E07000043","E07000044","E07000045","E07000046","E07000047","E10000013","E07000078","E07000079","E07000080","E07000081","E07000082","E07000083","E10000027","E07000187","E07000188","E07000246","E07000189"
```

Comment:
I believe we can shorten all these spatial_resolution above to:
`"E0", "E1", "E9"`

just like we have in some of the datasets that have been published to the staging platform - `datasets.json`